thrift/src/main/thrift/contentatom.thrift

```diff
@@ -54,6 +54,10 @@ struct ContentChangeDetails {
 
 struct Flags {
   1: optional bool suppressFurniture
+  2: optional bool sensitive
+  3: optional bool legallySensitive
+  4: optional bool suppressRelatedContent
+  5: optional bool blockAds
```

Comment:
@mchv will double check with @clloyd there were cards for all of them on the board so I put them all in but there may have been a discussion since the cards were made?